Source/WebKit/UIProcess/Inspector/mac/WKInspectorViewController.mm

```diff
@@ -100,6 +101,26 @@ - (void)dealloc
     return _delegate.getAutoreleased();
 }
 
+- (BOOL)isAttached
```

Comment:
Same here, this should be auto-synthesized for free and thus you don't need to implement it.